engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/DefaultOAuth2ServiceFactory.scala

```diff
@@ -63,7 +44,7 @@ object DefaultOAuth2ServiceFactoryWithProfileFormat {
       throw new Exception("profileFormat is missing in the authentication configuration")
     } match {
       case ProfileFormat.GITHUB => new DefaultOAuth2ServiceFactoryWithProfileFormat[GitHubProfileResponse](GitHubProfile)
-      case ProfileFormat.AUTH0 => new DefaultOAuth2ServiceFactoryWithProfileFormat[Auth0ProfileResponse](Auth0Profile)
+      case ProfileFormat.DEFAULT => new DefaultOAuth2ServiceFactoryWithProfileFormat[DefaultProfileResponse](DefaultProfile)
```

Comment:
Why right now it's default? Isn't it still AUTH0?

Reply:
maybe explicitly OIDC?

Reply:
The default profile is meant to be in the format of `/userinfo` response of OIDC which is widely accepted. Auth0 delivers this endpoint too.